packages/docs/src/components/DocsContent.vue

```diff
@@ -37,12 +37,12 @@
       <MarkdownView
         v-else-if="block.type === BlockType.PARAGRAPH"
         :key="block.type + index"
-        :value="`${$t(block.translationString)}`"
+        :value="`${$tie(block.translationString)}`"
       />
       <MarkdownView
         v-else-if="block.type === BlockType.LIST"
         :key="block.type + index"
-        :value="`${block.translationStringList.map(translationString=>'- ' + $t(translationString))
+        :value="`${block.translationStringList.map(translationString=>'- ' + $tie(translationString))
```

Comment:
May be move it to method handler, it would be more good for code reading?

Reply:
Fixed

Reply:
I meant to do something like that:

```javascript
translateAndMark(translations: string[]): string {
  const tie = (this as any).$tie;

  return translations
    .map((t: string): string => `- ${tie(t)}`
    .join('\n');
}
```

And usage:

```javascript
:value="translateAndMark(block.translationStringList)"
```

Reply:
`this.$tie` should work fine.

So should be something like: 
```typescript
translateAndMark(translations: string[]): string {
  return translations
    .map((t: string): string => `- ${this.$tie(t)}`
    .join('\n');
}
```

Reply:
Thanks for the tips :)
I have corrected.
But my linter throws an error on `this.$tie`. So I have to apply `(this as any).$tie`

Reply:
It's strange warning for `this`. I think it's point for this [issue](https://github.com/epicmaxco/vuestic-ui/issues/1034).